packages/vaadin-lumo-styles/color.js

```diff
@@ -211,6 +211,9 @@ const color = css`
     background-color: var(--lumo-contrast-10pct);
     border-radius: var(--lumo-border-radius-m);
   }
+  pre code {
```

Comment:
I’m not sure there’s ever any other use case for having a `<code>` element inside a `<pre>` element – so take this as me being extra careful – but I wonder if this would be more robust, as most often this is what syntax highlighters output:

```suggestion
  pre[class^=language] code[class^=language] {
```

Reply:
I think in the case @samie, there is not syntax highlighting.

If we want to trow in syntax highlighting to this "game", which I think is kind of more advanced use case of this and should be handled separately, my guess would be that we should try to limit that original rule above this. Syntax hightlighters might well have their own background rules that might interfere these rules. But, this is pure speculation until we have some actual examples.

Reply:
But yeah, I'm not really a domain expert on CSS hacking anymore, feel free to do what you see best.

Reply:
This can be either `<pre><code>...` or `<pre><code class="language-ruby">...` but double background text colors still seem unlikely wanted. How to do a low-priority selector to clean it by default and still allow override? 

( Output is specified in [commonmark.org](https://commonmark.org/) there are more examples in https://spec.commonmark.org/0.31.2/#fenced-code-blocks  )